project/Dependencies.scala

```diff
@@ -132,7 +132,7 @@ object Dependencies {
   // FIXME
   val sdkScala = deps ++= coreDeps ++ Seq(jacksonScala)
 
-  val sdkScalaTestKit = deps ++= Seq(testContainers, logback % "test;provided", scalaTest % Test)
+  val sdkScalaTestKit = deps ++= Seq(testContainers, logback % "test;provided", scalaTest)
```

Comment:
Do we require scalatest for testing with the testkit? Isn't it just that we _use_ it ourselves (so not a transitive dependency by default, as before).

Reply:
Actually, this change is for `codegenScalaCompilationTest` which depends on `sdkScalaTestKit` where we want to compile the test code together with the main code. At least, it's how I understand it. A similar story was with the junit dependency in `Dependencies#sdkJavaTestKit`.

Reply:
Shouldn't we add the non test scoped ScalaTest dependency to that project instead in that case?

Reply:
Yeah, indeed. 

The java testkit has junit in provided scope, but it does have a dependency on teskit. The scala testkit production code has not dependency on scalatest at all.

Reply:
Actually, I don't need to change the scope of this dependency. Somehow it is there with the correct scope already.